db/migrate/20211117234907_remove_spent_type_from_groups.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSpentTypeFromGroups < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :groups, :spent_type, :boolean
+  end
+end
```

Comment:
Is this migration needed?

I don't see the `spent_type` added before

Reply:
We created the model but after to created it we analyze that it are not necessary for our logic, i forgot to remove this migration, i remove it right now, thanks

Reply:
Done @EduardoGHdez